internal/api/attestationconfigapi/fetcher.go

```diff
@@ -64,35 +65,45 @@ func newFetcherWithClientAndVerifier(client apifetcher.HTTPClient, cosignVerifie
 	return &fetcher{HTTPClient: client, verifier: cosignVerifier, cdnURL: url}
 }
 
-// FetchAzureSEVSNPVersionList fetches the version list information from the config API.
-func (f *fetcher) FetchAzureSEVSNPVersionList(ctx context.Context, attestation AzureSEVSNPVersionList) (AzureSEVSNPVersionList, error) {
+// FetchSEVSNPVersionList fetches the version list information from the config API.
+func (f *fetcher) FetchSEVSNPVersionList(ctx context.Context, list SEVSNPVersionList) (SEVSNPVersionList, error) {
 	// TODO(derpsteb): Replace with FetchAndVerify once we move to v2 of the config API.
```

Comment:
Verifying `list.sig` would require that we consistently create the signature with all operations that modify the list of versions. Since v1 does not dictate the existence of a `list.sig` file there might be clients that update `list` without `list.sig`. Those clients are valid for v1. That update from a v1 client in turn would break clients that verify `list.sig`.

Reply:
good catch